ros2cli/ros2cli/node/daemon.py

```diff
@@ -100,8 +100,9 @@ class DaemonNode:
 
     def __init__(self, args):
         self._args = args
+        host = '127.0.0.1' if platform.system() == 'Windows' else 'localhost'
```

Comment:
So if we are going to follow in the same idea as what Rolling has, I'll suggest we just unconditionally use 127.0.0.1 here.